go/tasks/plugins/k8s/pod/plugin.go

```diff
@@ -75,7 +78,16 @@ func (p plugin) BuildResource(ctx context.Context, taskCtx pluginsCore.TaskExecu
 	return pod, nil
 }
 
-func (plugin) GetTaskPhase(ctx context.Context, pluginContext k8s.PluginContext, r client.Object) (pluginsCore.PhaseInfo, error) {
+func (p plugin) GetTaskPhase(ctx context.Context, pluginContext k8s.PluginContext, r client.Object) (pluginsCore.PhaseInfo, error) {
+	logPlugin, err := logs.InitializeLogPlugins(logs.GetLogConfig())
```

Comment:
We should probably do this once when we create the plugin and just keep a pointer to the logPlugin...